src/transformers/modeling_tf_utils.py

```diff
@@ -1528,8 +1536,8 @@ def train_step(self, data):
         output_to_label = {val: key for key, val in label_to_output.items()}
         if not self._using_dummy_loss and parse(tf.__version__) < parse("2.11.0"):
             # Newer TF train steps leave this out
-            data = data_adapter.expand_1d(data)
-        x, y, sample_weight = data_adapter.unpack_x_y_sample_weight(data)
+            data = expand_1d(data)
+        x, y, sample_weight = tf.keras.utils.unpack_x_y_sample_weight(data)
```

Comment:
Has this been here for all the TF versions we support?

Reply:
Yes, since at least 2.4!